src/vma/iomux/epfd_info.cpp

```diff
@@ -334,10 +334,16 @@ int epfd_info::add_fd(int fd, epoll_event *event)
 		lock();
 
 		if (ret < 0) {
-			if (errno == EEXIST) {
-				__log_dbg("epoll_ctl: fd=%d is already registered with this epoll instance (%d)", fd, this->get_epoll_fd());
-			} else { // errno == ENOMEM
-				__log_dbg("epoll_ctl: fd=%d is already registered with another epoll instance (%d), cannot register to epoll (%d)", fd, temp_sock_fd_api->get_epoll_context_fd(), this->get_epoll_fd());
+			switch (errno) {
+			case EEXIST:
+				__log_dbg("epoll_ctl: fd=%d is already registered with this epoll instance (%d)", fd, m_epfd);
```

Comment:
@liranoz12  preferably add the "(errno=%d %m)" in all gbd_log prints (in all switch error cases).
Makes it easier to compare to application logged error message